GrailDiary/GrailDiary/ViewControllers/POIsTableViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  POIsTableViewController.swift
+//  GrailDiary
+//
+//  Created by Cora on 5/21/20.
+//  Copyright © 2020 Cora. All rights reserved.
+//
+
+import UIKit
+
+class POIsTableViewController: UIViewController {
+
+    var POIs: [POI] = [POI(location: "Sedona, Arizona", country: "USA", clues: ["Beautiful", "Great Rocks", "Inspiring"]), POI(location: "Alameda, California", country: "USA", clues: ["Hometown"])]
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "addPOIModalSegue" {
+            if let addPOIVC = segue.destination as? AddPOIViewController {
+                addPOIVC.delegate = self
+            }
+        }
+        else if segue.identifier == "showPOIDetailSegue" {
+            if let indexPath = tableView.indexPathForSelectedRow,
+                let poiDetailVC = segue.destination as? POIDetailViewController {
+                poiDetailVC.poi = POIs[indexPath.row]
+            }
+        }
+    }
+}
+
+extension POIsTableViewController: UITableViewDelegate {
+}
```

Comment:
Might consider deleting this extension if it's not going to be used, though I realize you would also have to not set your vc as the delegate. I can also see the point of view of leaving it in for future expansion...